bin/utils/subkey/README.md

```diff
@@ -85,11 +87,27 @@ output:
 
 ### Inspecting a key
 
-You can inspect a given URI (mnemonic, seed, public key, or address) and recover the public key and the address.
+If you have *some data* about a key, `subkey inpsect` will help you discover more information about this key.
+
+If you have **secrets** that you would like to verify for instance, you can use:
+
+    subkey inspect < mnemonic | seed >
+
+If you have only **public data**, you can see a subset of the information:
 
-    subkey inspect <mnemonic | seed | pubkey | address>
+    subkey inspect --public < pubkey | address >
 
-*Example Output*:
+**NOTE**: While you will be able to recover the secret seed from the mnemonic, the opposite is not possible.
```

Comment:
Oh interesting, TIL!

Reply:
Actually, it *may* be possible but `subkey` does not allow it currently.